### PR TITLE
fix some mistake, delete some repetitive committment

### PR DESCRIPTION
**Pull request by zkdpower**
fix some mistake, delete some repetitive committment

### CODE REVIEW

File: src/duckstation-qt/translations/duckstation-qt_zh-cn.ts [hunk 1]
```diff
@@ -3638,7 +3634,7 @@ The saves will not be recoverable.</source>
     <message>
         <location filename="../qtprogresscallback.cpp" line="10"/>
         <source>DuckStation</source>
-        <translation>DuckStation</translation>
+        <translation></translation>
```
Comment (stenzek) [inline src/duckstation-qt/translations/duckstation-qt_zh-cn.ts:3637]: Same here

File: src/duckstation-qt/translations/duckstation-qt_zh-cn.ts [hunk 2]
```diff
@@ -36,7 +36,7 @@
     <message>
         <location filename="../aboutdialog.cpp" line="34"/>
         <source>License</source>
-        <translation>License</translation>
+        <translation></translation>
```
Comment (stenzek) [inline src/duckstation-qt/translations/duckstation-qt_zh-cn.ts:39]: Is removing this string intended?

File: src/duckstation-qt/translations/duckstation-qt_zh-cn.ts (hunk 1 above)
Comment (zkdpower) [reply src/duckstation-qt/translations/duckstation-qt_zh-cn.ts:3637]: i don't know programming, i want ask you if you source code here is "DuckStation",translation below is still reserved necessarily?

Comment (stenzek) [reply src/duckstation-qt/translations/duckstation-qt_zh-cn.ts:3637]: Seems that if `<translation></translation>` is blank, it uses the original string.

So, this is fine :)

Comment (zkdpower) [reply src/duckstation-qt/translations/duckstation-qt_zh-cn.ts:3637]: OK, all my translation cut is exactly same meaning to original string.

**Comment by zkdpower** [comment]
hello,stenzek,i want ask you how to find some subtitle resource code like "General, Graphics, Save States, Audio" in Hotkey Settings。besides,"Load or Save Game/Global 1~10" is also unable to find.[https://share.weiyun.com/PY82CIPL](url)[https://share.weiyun.com/S8UJ4mn2](url)

**Comment by stenzek** [comment]
> "General, Graphics, Save States, Audio" in Hotkey Settings

These should be translatable now. Run lupdate to add the strings to the .ts file (`src/duckstation-qt/update_translations.bat`).

> "Load or Save Game/Global 1~10"

Will be fixing these soon.

**Comment by zkdpower** [comment]
HI, honored author, you know, my previous chinese language online modification submitting is base on the Phen-nix's works，but now he want to quit the project. SO, how could i do to maintain it, what is iupdate and irelease？ i am a rookie without any programming technique. i have downloaded the latest Qt 5.15.




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Connor McLaughlin"<notifications@github.com&gt;; 
发送时间: 2020年10月4日(星期天) 晚上7:15
收件人: "stenzek/duckstation"<duckstation@noreply.github.com&gt;; 
抄送: "游戏音乐种花"<31569706@qq.com&gt;; "Author"<author@noreply.github.com&gt;; 
主题: Re: [stenzek/duckstation] fix some mistake, delete some repetitive committment (#921)





  
"General, Graphics, Save States, Audio" in Hotkey Settings
  
These should be translatable now. Run lupdate to add the strings to the .ts file (src/duckstation-qt/update_translations.bat).
  
"Load or Save Game/Global 1~10"
  
Will be fixing these soon.
 
&mdash;

**Comment by zkdpower** [comment]
would you like to consider supporting the "ISO,PBP,CCD,MDS" format image file next?&nbsp;



------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Connor McLaughlin"<notifications@github.com&gt;; 
发送时间: 2020年10月4日(星期天) 晚上7:15
收件人: "stenzek/duckstation"<duckstation@noreply.github.com&gt;; 
抄送: "游戏音乐种花"<31569706@qq.com&gt;; "Author"<author@noreply.github.com&gt;; 
主题: Re: [stenzek/duckstation] fix some mistake, delete some repetitive committment (#921)





  
"General, Graphics, Save States, Audio" in Hotkey Settings
  
These should be translatable now. Run lupdate to add the strings to the .ts file (src/duckstation-qt/update_translations.bat).
  
"Load or Save Game/Global 1~10"
  
Will be fixing these soon.
 
&mdash;

**Comment by zkdpower** [comment]
would you like it to consider supporting the "ISO,PBP,CCD,MDS" format image file next? 



------------------ 原始邮件 ------------------
发件人: "Connor McLaughlin"<notifications@github.com&gt;; 
发送时间: 2020年10月4日(星期天) 晚上7:15
收件人: "stenzek/duckstation"<duckstation@noreply.github.com&gt;; 
抄送: "游戏音乐种花"<31569706@qq.com&gt;; "Author"<author@noreply.github.com&gt;; 
主题: Re: [stenzek/duckstation] fix some mistake, delete some repetitive committment (#921)





  
"General, Graphics, Save States, Audio" in Hotkey Settings
  
These should be translatable now. Run lupdate to add the strings to the .ts file (src/duckstation-qt/update_translations.bat).
  
"Load or Save Game/Global 1~10"
  
Will be fixing these soon.

**Comment by zkdpower** [comment]
would you consider it to support the "ISO,PBP,CCD,MDS" format image file next? 



------------------ 原始邮件 ------------------
发件人: "Connor McLaughlin"<notifications@github.com&gt;; 
发送时间: 2020年10月4日(星期天) 晚上7:15
收件人: "stenzek/duckstation"<duckstation@noreply.github.com&gt;; 
抄送: "游戏音乐种花"<31569706@qq.com&gt;; "Author"<author@noreply.github.com&gt;; 
主题: Re: [stenzek/duckstation] fix some mistake, delete some repetitive committment (#921)





  
"General, Graphics, Save States, Audio" in Hotkey Settings
  
These should be translatable now. Run lupdate to add the strings to the .ts file (src/duckstation-qt/update_translations.bat).
  
"Load or Save Game/Global 1~10"
  
Will be fixing these soon.

**Comment by zkdpower** [comment]
i find iupdate.exe and irelease.exe in&nbsp;C:\Qt\5.15.1\android\bin folder




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Connor McLaughlin"<notifications@github.com&gt;; 
发送时间: 2020年10月4日(星期天) 晚上7:15
收件人: "stenzek/duckstation"<duckstation@noreply.github.com&gt;; 
抄送: "游戏音乐种花"<31569706@qq.com&gt;; "Author"<author@noreply.github.com&gt;; 
主题: Re: [stenzek/duckstation] fix some mistake, delete some repetitive committment (#921)





  
"General, Graphics, Save States, Audio" in Hotkey Settings
  
These should be translatable now. Run lupdate to add the strings to the .ts file (src/duckstation-qt/update_translations.bat).
  
"Load or Save Game/Global 1~10"
  
Will be fixing these soon.
 
&mdash;

**Comment by stenzek** [comment]
You don't need to install Qt, I include all the necessary programs in the repository.

1. Install Git if you don't already have it.
2. Clone the repository with submodules `git clone --recursive https://github.com/stenzek/duckstation.git`

You only have to do this once. In the future:
1. Reset any local changes with `git reset --hard`. If you have modified the ts file you may wish to back it up first, as this will remove any local changes.
2. Open `src\duckstation-qt`
3. Run `update_translations.bat` - this will add new strings (it's a wrapper around lupdate).
4. Open Linguist with `dep\msvc\qt\5.15.0\msvc2017_64\bin\linguist.exe`.
5. Follow the instructions in "Translating strings" in https://github.com/stenzek/duckstation/wiki/Translating-to-new-languages
6. Submit the updated .ts file as a pull request

**Comment by zkdpower** [comment]
To update an existing translation (Windows):

Update your Git repository by running&nbsp;git pull. If it gives you an error about local changes, run&nbsp;git reset --hard&nbsp;and&nbsp;git clean -d -f -x&nbsp;to clear them out, and run&nbsp;git pull&nbsp;again. But make sure you have nothing you want to save in the repo first, I won't take responsibility if you delete your work this way.

Run the&nbsp;update-and-edit-language.bat&nbsp;file in&nbsp;src\duckstation-qt\translations. This will add the new strings to the&nbsp;.ts&nbsp;file and report how many new strings there are for translation, and open it in Qt Linguist for editing.
hi,stenzek,i want to know what is&nbsp;"git pull"，"git rest --hard" ，"git clean -d -f -x"，where to input these commands?&nbsp;





------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "stenzek/duckstation"                                                                                    <notifications@github.com&gt;;
发送时间:&nbsp;2020年10月5日(星期一) 中午1:21
收件人:&nbsp;"stenzek/duckstation"<duckstation@noreply.github.com&gt;;
抄送:&nbsp;"游戏音乐种花"<31569706@qq.com&gt;;"Comment"<comment@noreply.github.com&gt;;
主题:&nbsp;Re: [stenzek/duckstation] fix some mistake, delete some repetitive committment (#921)





 
You don't need to install Qt, I include all the necessary programs in the repository.
  
Install Git if you don't already have it.
 
Clone the repository with submodules git clone --recursive https://github.com/stenzek/duckstation.git
  
You only have to do this once. In the future:
  
Reset any local changes with git reset --hard. If you have modified the ts file you may wish to back it up first, as this will remove any local changes.
 
Open src\duckstation-qt
 
Run update_translations.bat - this will add new strings (it's a wrapper around lupdate).
 
Open Linguist with dep\msvc\qt\5.15.0\msvc2017_64\bin\linguist.exe.
 
Follow the instructions in "Translating strings" in https://github.com/stenzek/duckstation/wiki/Translating-to-new-languages
 
Submit the updated .ts file as a pull request

**Comment by stenzek** [comment]
In Git Bash, after cd'ing to the duckstation directory.

**Comment by zkdpower** [comment]
what is git bash?



------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Connor McLaughlin"<notifications@github.com&gt;; 
发送时间: 2020年10月5日(星期一) 晚上9:42
收件人: "stenzek/duckstation"<duckstation@noreply.github.com&gt;; 
抄送: "游戏音乐种花"<31569706@qq.com&gt;; "Comment"<comment@noreply.github.com&gt;; 
主题: Re: [stenzek/duckstation] fix some mistake, delete some repetitive committment (#921)





 
In Git Bash, after cd'ing to the duckstation directory.
 
&mdash;

**Comment by zkdpower** [comment]
my clone repository position&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; C:\Users\ZKD\Documents\GitHub\duckstation\
my existing language position&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; C:\Users\ZKD\Documents\GitHub\duckstation\src\duckstation-qt\translations\duckstation-qt_zh-cn.ts
my linguist position&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;C:\Qt\5.15.1\msvc2019_64\bin\linguist.exe
my lupdate positon&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;C:\Qt\5.15.1\msvc2019_64\bin\lupdate.exe
my lrelease positon&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;C:\Qt\5.15.1\msvc2019_64\bin\lrelease.exe
……


because the Phen-nix's has finished previous version, so i only need to update an existing translation now, how should i do next?


------------------ 原始邮件 ------------------
发件人:                                                                                                                        "stenzek/duckstation"                                                                                    <notifications@github.com&gt;;
发送时间:&nbsp;2020年10月5日(星期一) 中午1:21
收件人:&nbsp;"stenzek/duckstation"<duckstation@noreply.github.com&gt;;
抄送:&nbsp;"游戏音乐种花"<31569706@qq.com&gt;;"Comment"<comment@noreply.github.com&gt;;
主题:&nbsp;Re: [stenzek/duckstation] fix some mistake, delete some repetitive committment (#921)





 
You don't need to install Qt, I include all the necessary programs in the repository.
  
Install Git if you don't already have it.
 
Clone the repository with submodules git clone --recursive https://github.com/stenzek/duckstation.git
  
You only have to do this once. In the future:
  
Reset any local changes with git reset --hard. If you have modified the ts file you may wish to back it up first, as this will remove any local changes.
 
Open src\duckstation-qt
 
Run update_translations.bat - this will add new strings (it's a wrapper around lupdate).
 
Open Linguist with dep\msvc\qt\5.15.0\msvc2017_64\bin\linguist.exe.
 
Follow the instructions in "Translating strings" in https://github.com/stenzek/duckstation/wiki/Translating-to-new-languages
 
Submit the updated .ts file as a pull request

**Comment by stenzek** [comment]
Git bash provides a shell for the command line version of Git.

I haven't used it, but you can probably use "Revert all changes" in github desktop to accomplish the same as the reset commands.

To add the new strings to the ts file, run `update_translations.bat` in `C:\Users\ZKD\Documents\GitHub\duckstation\src\duckstation-qt`.

**Comment by zkdpower** [comment]
could record a video teach me?i am chaos.






------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "游戏音乐种花"                                                                                    <31569706@qq.com&gt;;
发送时间:&nbsp;2020年10月6日(星期二) 凌晨2:00
收件人:&nbsp;"stenzek/duckstation"<reply@reply.github.com&gt;;"stenzek/duckstation"<duckstation@noreply.github.com&gt;;
抄送:&nbsp;"Author"<author@noreply.github.com&gt;;
主题:&nbsp;回复： [stenzek/duckstation] fix some mistake, delete some repetitive committment (#921)












------------------ 原始邮件 ------------------
发件人: "Connor McLaughlin"<notifications@github.com&gt;; 
发送时间: 2020年10月4日(星期天) 晚上7:15
收件人: "stenzek/duckstation"<duckstation@noreply.github.com&gt;; 
抄送: "游戏音乐种花"<31569706@qq.com&gt;; "Author"<author@noreply.github.com&gt;; 
主题: Re: [stenzek/duckstation] fix some mistake, delete some repetitive committment (#921)





  
"General, Graphics, Save States, Audio" in Hotkey Settings
  
These should be translatable now. Run lupdate to add the strings to the .ts file (src/duckstation-qt/update_translations.bat).
  
"Load or Save Game/Global 1~10"
  
Will be fixing these soon.

**Comment by zkdpower** [comment]
------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Connor McLaughlin"<notifications@github.com&gt;; 
发送时间: 2020年10月4日(星期天) 晚上7:15
收件人: "stenzek/duckstation"<duckstation@noreply.github.com&gt;; 
抄送: "游戏音乐种花"<31569706@qq.com&gt;; "Author"<author@noreply.github.com&gt;; 
主题: Re: [stenzek/duckstation] fix some mistake, delete some repetitive committment (#921)





  
"General, Graphics, Save States, Audio" in Hotkey Settings
  
These should be translatable now. Run lupdate to add the strings to the .ts file (src/duckstation-qt/update_translations.bat).
  
"Load or Save Game/Global 1~10"
  
Will be fixing these soon.
 
&mdash;

**Comment by zkdpower** [comment]
ok,i made it.delete all,using your explaining way in email, before now ,i always use the github desktop soft to fetch and pull always failure,even the right steps ,always generate the different ts.template.




------------------ 原始邮件 ------------------
发件人:                                                                                                                        "stenzek/duckstation"                                                                                    <notifications@github.com&gt;;
发送时间:&nbsp;2020年10月5日(星期一) 中午1:21
收件人:&nbsp;"stenzek/duckstation"<duckstation@noreply.github.com&gt;;
抄送:&nbsp;"游戏音乐种花"<31569706@qq.com&gt;;"Comment"<comment@noreply.github.com&gt;;
主题:&nbsp;Re: [stenzek/duckstation] fix some mistake, delete some repetitive committment (#921)





 
You don't need to install Qt, I include all the necessary programs in the repository.
  
Install Git if you don't already have it.
 
Clone the repository with submodules git clone --recursive https://github.com/stenzek/duckstation.git
  
You only have to do this once. In the future:
  
Reset any local changes with git reset --hard. If you have modified the ts file you may wish to back it up first, as this will remove any local changes.
 
Open src\duckstation-qt
 
Run update_translations.bat - this will add new strings (it's a wrapper around lupdate).
 
Open Linguist with dep\msvc\qt\5.15.0\msvc2017_64\bin\linguist.exe.
 
Follow the instructions in "Translating strings" in https://github.com/stenzek/duckstation/wiki/Translating-to-new-languages
 
Submit the updated .ts file as a pull request

**Pull request merged**